dtmi/himax/accelerometers-1.json

```diff
@@ -0,0 +1,30 @@
+{
+"@context": "dtmi:dtdl:context;2",
+"@id": "dtmi:himax:accelerometers;1",
+"@type": "Interface",
+"displayName": "Accelerometers",
+"description": "Accelerometers function in AIoT-NB-G2.",
+"contents": [
+{
+"@type": "Telemetry",

```

Comment:
Consider using a complex type for acceleration to use a single Telemetry element